utils/openebs-cleanup.yml

```diff
@@ -34,8 +34,27 @@
   args:
     executable: /bin/bash
 
+- name: Check the openebs components are still there or not
+  shell: kubectl get pods -n {{ namespace }} --no-headers | awk '{print $1}' | wc -l
+  args:
+    executable: /bin/bash
+  register: pod_list
+  until: "pod_list.stdout=='0'"
+  retries: 20
+  delay: 10
+
 ## Cleaning up OpenEBS Namespace
 - name: Removing openebs namespace
   shell: kubectl delete ns {{ namespace }}
   args:
-    executable: /bin/bash        
\ No newline at end of file
+    executable: /bin/bash
+
+## Checking openebs ns is deleted or not
+- name: Checking openebs ns deleted or not.
```

Comment:
A better task name would be - `Waiting for graceful termination of openebs namespace`